### ISSUE TITLE
ERROR admin:2442 - editProperties - unable to load the section form definition for endpoint=apps/local/splunk-add-on-jira-alerts

### ISSUE THREAD

**Issue by sdneil** [splunk/splunk-add-on-jira-alerts#6]
Has this error been seen & overcome before?

`ERROR admin:2442 - editProperties - unable to load the section form definition for endpoint=apps/local/splunk-add-on-jira-alerts
`
We verified the username and password in curl to the API for, GET (listProjects) and POST (createIssue) no issues.  So we think we have the pre-requisites set up.

**Comment by vlours** [comment]
Hi @sdneil,

Usually Splunk Applications are located in: etc/apps/<application>, not apps/local/<application>

You should get this application working if stored in etc/apps/splunk-add-on-jira-alerts
Once the setup completed, you should get a new 'local' folder in the application including your custom settings.

I hope this will help you.

Cheers,

**Comment by pdudhaiya-crest** [comment]
I don't think this is related to the path of the app or addon. I have the add-on under $SPLUNK_HOME/etc/apps and I am facing the same issue
